crunchydb/high-availablility/PostgresCluster.yaml

```diff
@@ -22,7 +23,10 @@ spec:
   #         limits:
   #           cpu: 100m
   #           memory: 64Mi
-  postgresVersion: 13
+  postgresVersion: 15
+  postGISVersion: "3.3"
+  image: artifacts.developer.gov.bc.ca/bcgov-docker-local/crunchy-postgres-gis:ubi8-15.2-3.3-0
```

Comment:
```suggestion
```

This can actually be removed now, as the operator itself has been updated to provide an appropriate default image from artifactory!